spec/requests/api/providers_spec.rb

```diff
@@ -1116,4 +1116,14 @@ def gen_import_request
       end
     end
   end
+
+  context "#options" do
+    it "returns options for queried provider class" do
+      api_basic_authorize
```

Comment:
@imtayadeway I am not sure how to remove this requirement. I also see this in many other examples: cluster, hosts, authentications. How do you suggest solving this?